### PR TITLE
Updates after repository transfer

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: .github/workflows/ci-tests.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  tests:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # Check out pull request's HEAD commit instead of the merge commit to
+          # work-around an issue where wrong a commit is being checked out.
```
Comment (ptrus) [inline .github/workflows/ci-tests.yml:26]:
```suggestion
          # work-around an issue where wrong commit is being checked out.
```

**Review by ptrus — APPROVED** [review]
Should there also be an action for publishing to npm? (or will that be done manually?)

File: .github/workflows/ci-tests.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+jobs:
+  tests:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
```
Comment (tjanez) [inline .github/workflows/ci-tests.yml:24]: I think it's safe to remove this work-around. It has been fixed for a while: https://github.com/actions/checkout/issues/237#issuecomment-667431246.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
-[![CircleCI](https://circleci.com/gh/ZondaX/ledger-oasis-js/tree/master.svg?style=shield)](https://circleci.com/gh/ZondaX/ledger-oasis-js/tree/master)
+[![CI test status][github-ci-tests-badge]][github-ci-tests-link]
 
-This package provides a basic client library to communicate with the Oasis App running in a Ledger Nano S/X
+<!-- markdownlint-disable line-length -->
+[github-ci-tests-badge]: https://github.com/oasisprotocol/ledger-js/workflows/ci-tests/badge.svg
+[github-ci-tests-link]: https://github.com/oasisprotocol/ledger-js/actions?query=workflow:ci-tests+branch:master
+<!-- markdownlint-enable line-length -->
+
+This package provides a basic client library to communicate with the Oasis apps running in a Ledger Nano S/X.
```
Comment (tjanez) [inline README.md:11]:
```suggestion
This package provides a basic client library to communicate with the Oasis apps running on a Ledger Nano S/X.
```

**Comment by tjanez** [comment]
> Should there also be an action for publishing to npm? (or will that be done manually?)

@ptrus, did you just volunteer for that :slightly_smiling_face:?

**Review by tjanez — APPROVED** [review]
Other than that, it looks good!

File: .github/workflows/ci-tests.yml (hunk 2 above)
Comment (kostko) [reply .github/workflows/ci-tests.yml:24]: But this is still not closed https://github.com/actions/checkout/issues/299?

**Comment by kostko** [comment]
> Should there also be an action for publishing to npm? (or will that be done manually?)

Also for the release process, but let's do that separately once we introduce a proper release process (will open an issue).

File: .github/workflows/ci-tests.yml (hunk 2 above)
Comment (tjanez) [reply .github/workflows/ci-tests.yml:24]: Yea, but it has been indicated that it is a duplicate of https://github.com/actions/checkout/issues/237 and I think it should be closed.

**Pull request merged**